src/OrchardCore.Modules/OrchardCore.ContentFields/Fields/DateTimeField.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+using OrchardCore.ContentManagement;
+
+namespace OrchardCore.ContentFields.Fields
+{
+    public class DateTimeField : ContentField
+    {
+        public DateTime? Value { get; set; }
```

Comment:
I understand that a field could not be required and have a null value. But I don't understand how the current code would support that. Does it work if nothing is set in the editor, does the updater correctly assigns null?

Also related, let's say I set a value and save, and the value is not required, can I erase the value somehow?

Reply:
I was able to reset the value to nothing using the local editor.